CHANGELOG/v1.11.0-beta.0.md

```diff
@@ -0,0 +1,309 @@
+🚨 This is a BETA RELEASE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+<details>
+<summary>More details about the release</summary>
+
+:warning: **BETA RELEASE NOTES** :warning:
```

Comment:
The Changes since v1.11.0-alpha.1 section was nice on previous release notes. Can we also have that for beta? (not a blocker if the release note tool cannot do it right now)